code401-challenges/src/main/java/code401/challenges/ArrayShift.java

```diff
@@ -0,0 +1,20 @@
+package code401.challenges;
+
+public class ArrayShift {
+    public int[] insertShiftArray(int[] arr, int value){
+        int length = arr.length;
```

Comment:
Since you only use this variable once, it might have been simpler to use set newArr to `[arr.length + 1]`